libs/crypto/include/crypto/mcl_dkg.hpp

```diff
@@ -41,34 +41,25 @@ using Generator      = bn::G2;
 using MessagePayload = byte_array::ConstByteArray;
 using CabinetIndex   = uint32_t;
 
-/**
- * Helper functions for computations used in the DKG
- */
-void   SetGenerator(Generator &group_g);
-void   SetGenerators(Generator &group_g, Generator &group_h);
-bn::G2 ComputeLHS(bn::G2 &tmpG, bn::G2 const &G, bn::G2 const &H, bn::Fr const &share1,
-                  bn::Fr const &share2);
-
-bn::G2 ComputeLHS(bn::G2 const &G, bn::G2 const &H, bn::Fr const &share1, bn::Fr const &share2);
-
-void UpdateRHS(uint32_t rank, bn::G2 &rhsG, std::vector<bn::G2> const &input);
-
-bn::G2 ComputeRHS(uint32_t rank, std::vector<bn::G2> const &input);
-
-void ComputeShares(bn::Fr &s_i, bn::Fr &sprime_i, std::vector<bn::Fr> const &a_i,
-                   std::vector<bn::Fr> const &b_i, uint32_t rank);
-
-bn::Fr ComputeZi(std::set<uint32_t> const &parties, std::vector<bn::Fr> const &shares);
-
-std::vector<bn::Fr> InterpolatePolynom(std::vector<bn::Fr> const &a, std::vector<bn::Fr> const &b);
-
-// For signatures
-Signature SignShare(MessagePayload const &message, PrivateKey const &x_i);
-
-bool VerifySign(PublicKey const &y, MessagePayload const &message, Signature const &sign,
-                Generator const &G);
+struct DkgOutput
+{
+  bn::G2              group_public_key;
```

Comment:
bn::G2 -> PublicKey ? etc.

Style: Functions first, then variables.

Did you mean for there to be two DkgOutputs structs in this PR?

Reply:
Fixed the style and bn::G2. The other DkgOutputs also contains qual, which the crypto function TrustedDealerGenerateKeys does not need to know about. I will rename the DkgOutputs in beacon_manager_tests to something else to avoid confusion.